setup.py

```diff
@@ -57,6 +57,7 @@
             "pytest-mpi",
             "torchvision",
             "mpi4py",
+            "scipy",
```

Comment:
The `scipy` package is used in the example and should be moved to under `examples`.